nikola/utils.py

```diff
@@ -113,57 +110,8 @@
 unichr = chr
 
 
-class ApplicationWarning(Exception):
-    pass
-
-
-class ColorfulStderrHandler(ColorizedStderrHandler):
-    """Stream handler with colors."""
-
-    _colorful = False
-
-    def should_colorize(self, record):
-        """Inform about colorization using the value obtained from Nikola."""
-        return self._colorful
-
-
-def get_logger(name, handlers=None):
-    """Get a logger with handlers attached."""
-    l = logbook.Logger(name)
-    l.handlers += STDERR_HANDLER
-    return l
-
-
-STDERR_HANDLER = [ColorfulStderrHandler(
```

Comment:
Almost all plugins I'm using now fail with
```
AttributeError: module 'nikola.utils' has no attribute 'STDERR_HANDLER'
```
when doing `utils.get_logger('xxx', utils.STDERR_HANDLER)`. Also grepping in https://github.com/getnikola/plugins/ shows tons of plugins using `STDERR_HANDLER`. I guess we should still provide a dummy `STDERR_HANDLER` so that existing code won't break.

I've tried adding
```
STDERR_HANDLER = None
```
somewhere in this file, and with that, all my blogs and pages run through fine.

Reply:
Fixed in commit 5cd3e140a.